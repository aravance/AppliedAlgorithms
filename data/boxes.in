1 7
SB....T
1 7
SB..#.T
7 11
###########
#T##......#
#.#.#..####
#....B....#
#.######..#
#.....S...#
###########
8 4
....
.##.
.#..
.#..
.#.B
.##S
....
###T
14 17
S................
#######.###B####.
#######.###.####.
.................
.######.###.####.
.######.##..####.
.######.##.####..
................#
######.########.#
######.....####.#
##########.####..
#................
#.########.######
#...............T
20 20
TS..................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
..................B.
....................
20 20
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
STB.................
1 3
SBT
20 20
TS..................
..................#.
....................
................#...
...............#....
..............#.....
.............#......
............#.......
...........#........
..........#.........
.........#..........
........#...........
.......#............
......#.............
.....#..............
....#...............
...#................
..#.................
..................B.
............#.......
20 20
TS.................#
..................#.
.................#..
................#...
...............#....
..............#.....
.............#......
............#.......
...........#........
..........#.........
.........#..........
........#...........
.......#............
......#.............
.....#..............
....#...............
...#................
..#...............B.
............#.......
....................
20 20
TS.................#
..................#.
.................#..
................#...
...............#....
..............#.....
.............#......
............#.......
...........#........
..........#.........
.........#..........
........#...........
.......#............
......#.............
.....#..............
....#...............
...#................
..#...............B.
.#..................
....................
5 7
#######
#..BS.T
#.###.#
#.....#
#######
13 13
......T......
.....#####...
.....#...#...
.....#.#.#...
.....#...#...
######.###...
#....B..S#...
#.#.##.#.#...
#...##...#...
######.###...
.............
.............
.............
16 13
......T......
.....#####...
.....#...#...
.....#.#.#...
.....#...#...
######.###...
#....B..S#...
#.#.##.#.#...
#...##...#...
######.###...
#.......#....
#.......#....
#.######.....
.............
.............
###########..
13 13
......T......
.....#####...
.....#...#...
.....#.#.#...
.....#...#...
######.###...
#....B..S#...
#.#.##.#.#...
#...##...#...
##########...
.............
.............
.............
5 7
.####.T
.......
.B####.
S......
.......
10 20
.........##.........
............#.......
#.#.#.##.#.........#
.........##..#######
...#B#..##.........S
..#......#...##.####
##..######..###...##
#..................#
#...################
#..................T
9 9
#########
####T####
####S####
####B####
####.####
###....##
###....##
#####..##
#########
20 20
.........#..........
.........#..........
.........#.........#
.........#...#######
.........#..........
.........#..........
.........######.###.
..............#....#
.........#.##.#.....
.........#....#.....
.........#.####....#
#####.T..#.###......
...........###.....#
..########.###......
....BS...#.###......
.........#.###......
##########.###...#..
#..................#
#...####........####
#........##.........
20 20
.........#..........
.........#..........
.........#.........#
.........#...#######
...#####.#..........
...#...#.#..........
...#.#.########.###.
...#..........#....#
...###...#.##.#.....
.....#...#....#.....
.....#...#.####....#
#####.T..#.###......
.............#.....#
..########.#.#......
....BS...#...#......
.........#.###......
##########.###...#..
#..................#
#...####........####
#........##.........
0 0
