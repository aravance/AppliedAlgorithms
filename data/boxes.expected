Maze #1
EEEEE

Maze #2
Impossible.

Maze #3
eennwwWWWWeeeeeesswwwwwwwnNN

Maze #4
swwwnnnnnneeesssSSS

Maze #5
eeeeeeeeeeeSSnneeeeessswwwwWWWWeeennnwwwwssSSSSnnnwwwwwwwsssseeeeeeEEEEEEEEwwwwnnenneeeeessswSSSSesWWWWWeeeennnnwwwwwwwwwsseeeesSSnwwwwwwwwwsseeeeeeeeEEEEEE

Maze #6
eeeeeeeeeeeeeeeessssssssssssssssssseNNNNNNNNNNNNNNNNNNenWWWWWWWWWWWWWWWWWW

Maze #7
neeesW

Maze #8
E

Maze #9
eeeeeeeeeeeeeeeeeessssssssssssssssssWWWWWWWWWWWWWWWWWWswNNNNNNNNNNNNNNNNNN

Maze #10
sssssssssssssssssseeeeeeeeeeseeeeeeeennWWWWWWWWnwSesWWWWWWWWWWswNNNNNNNNNNNNNNNNNN

Maze #11
Impossible.

Maze #12
esswwwwnneEEE

Maze #13
wwWWWsswwnnEEEEEsseennWesswwnNNNeennwwSSSSSSSSwsEEEEseNNNNNNNNNNNenWWWW

Maze #14
wwWWWsswwnnEEEEEsseennWesswwnNNNeennwwSSSSSSSSesWWWWWnwSSSwsEEEEEEEEEEneesswNNNNNNNNNNNNNNenWWWWW

Maze #15
Impossible.

Maze #16
eNwnEEEEEwwwwsseeeeenN

Maze #17
wwwwssswwwwwwwwwwwwnneNseennwWWWswNNseennwnwwsEEEEEEEEEEseeennwwSSSSesWnwSSesWWWWWWWeeeeeeennnnnwnwwsswsswwwwsSSwsEEEEEEEEEEEEEEEE

Maze #18
SSSwsEneesswNenWswNNNN

Maze #19
WWWswNNwnEEEEEEEEEwnnnnneessssSSSSSSwsEneesswNenWswNNNNNNNNNNseeennwwWWWWnwSSSS

Maze #20
WWWswNNwnEEEEEEEEEEsseennWesswwnNNNNNseeennwwWWWWswNwwnneeSSSSS
